### ISSUE TITLE
Error message if --temp_dir is missing

### ISSUE THREAD

**Issue by axgraf** [BrooksLabUCSC/flair#42]
Running the flair collapse with --temp_dir option fails with a very unspecific error message when the temporary folder does not exists:

> "Possible minimap2/samtools error, specify paths or make sure they are in $PATH"

Either could you just test if the file/folder exists or just report a more specific error message,  so other users might not run into the same issue as I struggled quite long to find the core of the problem as I was searching for problems with minimap2/samtools.

https://github.com/BrooksLabUCSC/flair/blob/a7d2927f5929153648f26461ec58bd3345855b8d/flair.py#L254

Thx Alex

**Comment by belgravia** [comment]
Thanks for catching that!! That feeling when you add something to fix an issue and just create more issues... I've now added a check to make sure that the temp_dir the user specifies exists, and if it doesn't exist already it will make that directory. I've also updated the error message to tell users to check that all their files and folders exist so the next person who runs into this hopefully won't have to be too confused about how to approach the error. Glad you were able to figure it out.
-Alison
